yml/dp02_dc2.yaml

```diff
@@ -7090,692 +7090,692 @@ tables:
     "@id": "#DiaObject.decl"
     datatype: double
     mysql:datatype: DOUBLE
-    description:
+    description: Declination coordinate of the position of the diaObject at time radecTai
   - name: diaObjectId
     "@id": "#DiaObject.diaObjectId"
     datatype: long
     mysql:datatype: BIGINT
-    description:
+    description: Unique id.
```

Comment:
Is it worth expanding on this to say "Unique identifier for this object" or something? Maybe it's obvious from context, so the description is mostly redundant with the name?